src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -3061,6 +3064,10 @@ inline void Reduce::set_reduce_dim_flags() {
         SET_DST_DIM_VALUE(1, process_dst_dims[0], 1, 1, 1);
     }
 
+    // Depthwise fusion can be computed like eltwise fusion without broadcast, if is_depthwise_compatible is true.
+    bool is_depthwise_compatible = dims_size > 1 && process_dst_dims[1] == OB * OC * OD * OH * OW;
+    jcp.fuse_broadcast = jcp.layout == ReduceLayoutType::reduce_ncsp && !is_depthwise_compatible;
```

Comment:
Is there any possibility that hide fuse_broadcase inside kernel? or your optimized implementation droped some info in jcp in kernel that can not know this information?

Reply:
Thanks Chenhu for the comment! As the information of dimensions is not available in kernel, so this fuse_broadcase is needed to be passed into kernel.